config/kubernetes/sig-testing/teams.yaml

```diff
@@ -28,26 +28,35 @@ teams:
     - timothysc
     - xiangpengzhao
     privacy: closed
-  sig-testing-pr-reviews:
-    description: sig-testing PR reviews
-    maintainers:
-    - fejta
-    - spiffxp
-    members:
-    - akutz
-    - amwat
-    - BenTheElder
-    - chases2
-    - cjwagner
-    - clarketm
-    - ixdy
-    - Katharine
-    - krzyzacy
-    - michelle192837
-    - mithrav
-    - mkumatag
-    - stevekuznetsov
-    privacy: closed
+    teams:
+      sig-testing-leads:
+        description: sig-testing chairs and technical leads
+        maintainers:
+        - spiffxp
+        members:
+        - BenTheElder
+        - stevekuznetsov
+        privacy: closed
+      sig-testing-pr-reviews:
+        description: sig-testing PR reviews
+        maintainers:
+        - fejta
+        - spiffxp
+        members:
+        - akutz
+        - amwat
+        - BenTheElder
+        - chases2
+        - cjwagner
+        - clarketm
+        - ixdy
+        - Katharine
```

Comment:
this list could probably do with some tidying?

Reply:
agreed, I wanted to do that as a followup